### PR TITLE
Bug/em2022 detect annotation toggle changes

### PR DESCRIPTION
**Pull request by benouaer**
### JIRA link (if applicable) ###
- https://tools.hmcts.net/jira/browse/EM-2022
- https://tools.hmcts.net/jira/browse/EM-2023

### Change description ###
- react to changes to annotation toggle
- enable overriding of toolbar buttons

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts [hunk 1]
```diff
@@ -35,6 +35,10 @@ export class CommentComponent implements OnChanges {
   @ViewChild('form') form: ElementRef;
   @ViewChild('textArea') textArea: ElementRef;
 
+  constructor() {
+    this.MAX_COMMENT_LENGTH = 48;
```
Comment (velimd) [inline projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts]: is this best practise, just want to know

File: projects/media-viewer/src/lib/media-viewer.component.ts [hunk 2]
```diff
@@ -71,23 +71,40 @@ export class MediaViewerComponent implements OnChanges, AfterContentInit {
...
     }
   }
 
   onLoadException(exception: ViewerException) {
     this.viewerException.emit(exception);
   }
 }
+
+enum SupportedContentTypes {
+  PDF = 'pdf',
```
Comment (velimd) [inline projects/media-viewer/src/lib/media-viewer.component.ts]: nice, i was always wondering why we never use an enum

File: projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts (hunk 1 above)
Comment (benouaer) [reply projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts]: the other way is not supported in typescript <3.1.0, which JUI use

**Comment by linusnorton** [comment]
Does it matter? Aren't they consuming compiled JS instead of raw TS?

**Pull request merged**